### PR TITLE
Ability to deploy to mvn central

### PR DESCRIPTION
**Pull request by dlecocq**

### CODE REVIEW

File: provision.sh [hunk 1]
```diff
@@ -20,3 +20,13 @@ sudo apt-get install -y autoconf bison build-essential git g++ libssl-dev libyam
...
+
+# For deploying
+mkdir -p /vagrant/.gnupg
+ln -s /vagrant/.gnupg/ /home/vagrant/.gnupg
+echo '\
+    eval $(gpg-agent --daemon --no-grab --write-env-file $HOME/.gpg-agent-info) \
+    export GPG_TTY=$(tty) \
+    export GPG_AGENT_INFO' >> ~/.bash_profile
+ln -s /vagrant/settings-security.xml /home/vagrant/.m2/settings-security.xml
+ln -s /vagrant/settings.xml /home/vagrant/.m2/settings.xml
```
Comment (b4hand) [inline provision.sh]: Trailing newline.

**Comment by b4hand** [comment]
LGTM, minus minor nit.

**Comment by dlecocq** [comment]
Comment addressed. Will merge when it passes.

**Comment by dlecocq** [comment]
Somehow Travis builds got turned off for this repo. Maybe when it switched from private to public? At any rate, they've been re-enabled.

**Pull request merged**